themes/inpycon2024/templates/index.html

```diff
@@ -1,61 +1,56 @@
-{% extends "base.html"%}
-
-{% block body %}
-  <!-- blog section  -->
-  <section class="bg-blog">
+{% extends "base.html"%} {% block body %}
+<!-- blog section  -->
+<section class="blog-banner py-5 py-md-4">
     <div class="container">
-      <div class="row pt-5 pb-5">
-        <div class="col-md-12 pb-5">
-          <h2 class="com-head mb-0 text-center text-md-start" data-aos="fade-right"  data-aos-duration="3000">Our Blogs</h2>
+        <div class="col-md-12 text-center">
+            <h2
+                class="title-2 text-center mb-0 mb-md-5 pt-0 pt-md-5 pb-5"
+                data-aos="fade-up"
+                data-aos-duration="1000">
+                Our Blogs
+            </h2>
         </div>
-        {% for article in articles_page.object_list %}
-        <div class="blog-sec pt-3 pb-3">
-          <div class="row pt-4 pb-4">
-            <div class="col-lg-8">
-              <a href="{{SITEURL}}/{{article.url}}">
-                <h3 class="blog-head">{{article.title}}</h3>
-              </a>
-              <p class="blog-para">{{article.summary}}</p>
-              <div class="readmore-blog"><a href="{{SITEURL}}/{{article.url}}">Read More <img src="{{ SITEURL }}/theme/images/arrow-blog.png" class="img-fluid ps-2"></a></div>
-            </div>
-            <div class="col-lg-4 d-none d-lg-block">
-              <time datetime="{{article.date}}" itemprop="datePublished">
-                <h4 class="blog-date">Published: <span class="fw-bold">{{ article.date.strftime("%B %d, %Y") }}</span></h4>
-              </time>
-              <h4 class="blog-date">Author: <span class="fw-bold">{{ article.author | default("PyCon India Content Team", true) }}</span></h4>
+    </div>
+</section>
+<!-- blog-list -->
+<section class="blog-list">
+    <div class="container">
+        <div class="row">
+            {% for article in articles_page.object_list %}
+            <div class="col-md-12 my-3">
+                <div class="bg-box">
+                    <div class="bg-tit">
+                        <h3>{{article.title}}</h3>
+                        <p>{{article.summary}}</p>
+                        <a href="{{SITEURL}}/{{article.url}}">Read More</a>
```

Comment:
I think it might be better to put the link on th eheading and just remove Read More? No strong feelings though.